### DISCUSSION TITLE
为什么paddleocr用在项目上会报：FileNotFoundError: [Errno 2] No such file or directory: './ppocr/utils/EN_symbol_dict.txt'

### DISCUSSION THREAD

**Discussion by 694578908** [PaddlePaddle/PaddleOCR#13425] [Q&A]
在自己的项目上传了这个参数rec_char_dict_path='./ppocr/utils/EN_symbol_dict.txt'就会报错，但是我在PaddlerOCR-relase-2.7.1里面运行就正常的
![image](https://github.com/user-attachments/assets/fbfcedd4-b0da-45f5-9e76-1163b3b90c0a)
![image](https://github.com/user-attachments/assets/2969aee7-f382-41a3-a55b-e696c5cf2e63)

**Comment by GreatV** [comment]
有的呀，你需要改成绝对路径。

https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppocr/utils/EN_symbol_dict.txt

**Comment by UserWangZz** [comment]
当前脚本执行的路径不是项目的根路径，需要在paddleocr/路径下执行即可
